src/components/editTestCase/EditTestCase.tsx

```diff
@@ -209,9 +209,7 @@ const EditTestCase = () => {
   const [discardDialogOpen, setDiscardDialogOpen] = useState(false);
   const { updateMeasure } = measureStore;
   const load = useRef(0);
-  const [canEdit, setCanEdit] = useState<boolean>(
-    checkUserCanEdit(measure?.createdBy, measure?.acls)
-  );
+  const canEdit = checkUserCanEdit(measure?.createdBy, measure?.acls);
```

Comment:
How is the state of CanEdit related to inconsistent tabbing in JSON editor? Is the inconsistent behavior only seen for measures that are not owned/shared with users?